package.json

```diff
@@ -53,6 +53,7 @@
     "@rollup/plugin-node-resolve": "^9.0.0",
     "@rollup/plugin-replace": "^2.2.1",
     "@types/jest": "^25.2.1",
+    "@types/lodash": "^4.14.161",
```

Comment:
This should be a devDep -- it's only used during testing and not part of actual builds